NickvisionTagger.Shared/Models/Configuration.cs

```diff
@@ -68,6 +80,9 @@ public class Configuration : ConfigurationBase
     /// </summary>
     public Configuration()
     {
+        WindowWidth = 800;
```

Comment:
900

Reply:
Nope. I thought it was 900x700 too because it's what was set in blueprint, but it was actually overwritten by `SetDefaultSize(800, 700);` in MainWindow, see revomed line 116

Reply:
> Nope. I thought it was 900x700 too because it's what was set in blueprint, but it was actually overwritten by `SetDefaultSize(800, 700);` in MainWindow, see revomed line 116

Ah interesting, ok!